### DISCUSSION TITLE
Classify with multiple sentences

### DISCUSSION THREAD

**Discussion by siccokoetsier** [ThilinaRajapakse/simpletransformers#1346] [Q&A]
Hi,

I am pretty much a beginner in NLP and I am trying to build a basic binary BERT-classifier based on new Dutch text data + a tagged sentiment value in the training set. Yet in my case, each text item consists of multiple sentences. Most online examples only treat one-lined cases. I have the impression that Bert is very sensitive to this and needs to be fed in a very specific way.

How can I train the model with multiple sentence items? I do find some reference to this problem in the context of [RepresentationModel](https://simpletransformers.ai/docs/text-rep-examples/) ('encode_sentences'), but it's very hard for me to apply and integrate this code into the general classifier examples with ClassificationModel (f.e. [here](https://pypi.org/project/simpletransformers/0.48.4/#minimal-start-for-binary-classification) and [here](https://www.philschmid.de/bert-text-classification-in-a-different-language) ).

Thanks for any hints! Feel free to contact me anytime. Keep up the good work with this excellent lib!

**Comment by luizniero** [comment]
Hello. I have suggest a code for this. You can find my code at: https://github.com/ThilinaRajapakse/simpletransformers/pull/1360
